lms/djangoapps/course_api/blocks/transformers/block_completion.py

```diff
@@ -14,7 +14,7 @@ class BlockCompletionTransformer(BlockStructureTransformer):
     Keep track of the completion of each block within the block structure.
     """
     READ_VERSION = 1
-    WRITE_VERSION = 1
+    WRITE_VERSION = 2
```

Comment:
Actually, this seems to be a backward-compatible change, so we can revert bumping this version.

Reply:
reverted